statvar_imports/us_urban_school/ap_ib_gt_enrollment/download_ap_ib_gt.py

```diff
@@ -0,0 +1,461 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import requests
+import zipfile
+import io
+import pandas as pd
+import re
+from absl import logging, app, flags
+from pathlib import Path
+from datetime import date
+from retry import retry
+import sys
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_boolean('ap', False, 'Download Advanced Placement data only.')
+flags.DEFINE_boolean('ib', False, 'Download International Baccalaureate data only.')
+flags.DEFINE_boolean('gt', False, 'Download Gifted and Talented data only.')
+
+BASE_URL_TEMPLATE = "https://civilrightsdata.ed.gov/assets/ocr/docs/{}.zip"
+
+DATA_CONFIGS = {
+
+    # Advanced Placement Data Keywords
+
+    "ap": {
+
+        "keywords": [r'AP', r'Advanced Placement'],
+
+        "search_constraint": {
+
+            "2009-10": "CRDC-data_SCH_Pt 1-Advanced Placement.xlsx"
+
+        },
+
+        "output_name_fragment": "AP_Enrollment",
+
+        "output_dir_name": "advanced_placements/input_files",
+
+        "pvmap_config": "ap_enrollment_pvmap.csv"
+
+    },
+
+    # International Baccalaureate Data Keywords
+
+    "ib": {
+
+        "keywords": [r'IB', r'International Baccalaureate'],
+
+        "search_constraint": {
+
+            "2009-10": "CRDC-data_SCH_Pt 1-Enrollment_Pt 1-Students enrolled in IB Diploma Programme.xlsx"
+
+        },
+
+        "output_name_fragment": "IB_Enrollment",
+
+        "output_dir_name": "international_baccalaureate/input_files",
+
+        "pvmap_config": "ib_enrollment_pvmap.csv"
+
+    },
+
+    # Gifted and Talented Data Keywords
+
+    "gt": {
+
+        "keywords": [r'Gifted', r'Talented'],
+
+        "search_constraint": {
+
+            "2009-10": "CRDC-data_SCH_Pt 1-Enrollment_Pt 1-Students enrolled in Gifted-Talented Programs.xlsx"
+
+        },
+
+        "output_name_fragment": "GT_Enrollment",
+
+        "output_dir_name": "gifted_and_talented/input_files",
+
+        "pvmap_config": "gt_enrollment_pvmap.csv"
+
+    }
+
+}
+
```

Comment:
@HarishC727 Similar to previous review comment, this file contains a lot of unnecessary empty lines. Please do a cleanup of this file (and others) and remove unnecessary blank lines.